modules/ROOT/pages/prerequisites/index.adoc

```diff
@@ -134,9 +134,9 @@ It is assumed in the examples above, that on avarage, the original change was 4
 
 Storing meta-data in memory is important with respect to access and synchronization performance. oCIS has the concept of _spaces_ which you can see in a nutshell as a space containing files and folders which is not owned by a user by default, but gets assigned to users. Spaces can be the users home, any mount created or an unlimited quantity of shares and re-shares.
 
-Each space provides a virtual root and occupies 4K for this root meta-data. Additionally each space provides the complete meta-data information about the underlying content. As you can see, the quantity of meta-data can grow large.
+Each space provides a virtual root and occupies at least one meta-data block at its root. Additionally each space provides the complete meta-data information about the underlying content. As you can see, the quantity of meta-data can grow large.
 
-Whenever a file or folder (node) changes its content or metadata somewhere in the path, the ETag gets recalculated and the change is propagated to the spaces root for identifying a change. To let a client detect changes, the discovery phase starts at the spaces root and checks if the ETag has changed since the last discovery going forward in the tree of changed ETags.
+Whenever a file or folder (node) changes its content or meta-data somewhere in the path, the ETag gets recalculated and the change is propagated to the spaces root for identifying a change. To let a client detect changes, the discovery phase starts at the spaces root and checks if the ETag has changed since the last discovery going forward in the tree of changed ETags.
```

Comment:
metadata without the dash is actually correct: https://www.merriam-webster.com/dictionary/metadata

Reply:
ah, ok, will fix 👍

Reply:
fixed